tools/webpack-core/src/plugins/CopyBentleyStaticResourcesPlugin.ts

```diff
@@ -58,10 +58,7 @@ async function tryCopyDirectoryContents(source: string, target: string) {
   }
 }
 
-/** Prefer use of CopyStaticAssetsPlugin instead.
- * @note Will be removed in 3.0
- * @deprecated
- */
+/** Prefer use of CopyStaticAssetsPlugin instead when outside monorepos. */
```

Comment:
is this not deprecated anymore?

Reply:
We found an issue after deprecating this for the new plugin where assets in monorepos dont get copied, so we've been using this, and can't remove in 3.0. I thought since we're kinda stuck with this and theres no perfect alternative to remove the deprecation tag. Im being too wishy washy with this i guess. Can re-add the deprecation tag

Reply:
well if we're still going to remove it at a later date, I'd just remove the comment about 3.0... I guess that means we'll have to keep it until 4.0 but at least we can give people a warning not to use something they're not supposed to use?

Reply:
Yea, lets just remove the comment about this plugin being removed at all and if/when we fix the new one we can add it back as deprecated for possible removal in 4.0. Plus we can start transitioning all of our default tooling to use it (backend webpack tools and CRA).

Reply:
Lol i guess we dont publish docs for this pkg, so adding the deprecated tag previously didnt help anyone

Reply:
resolving comment to unblock pr from completing